www/privoxy/Portfile

```diff
@@ -300,15 +300,36 @@ variant https_inspection \
         system -W ${destroot}${tls_ca_dir} \
             "${prefix}/share/apple-pki-bundle/bin/pems_that_wont_expire_soon.sh \
                 ${prefix}/share/apple-pki-bundle/apple-pki-bundle.pem \
-                        >> trustedCAs.pem-temp"
-        copy ${destroot}${tls_ca_dir}/trustedCAs.pem-temp \
-            ${destroot}${tls_ca_dir}/trustedCAs.pem
+                        >> ${destroot}${tls_ca_dir}/trustedCAs.pem-temp"
+        copy        ${destroot}${tls_ca_dir}/trustedCAs.pem-temp \
+                    ${destroot}${tls_ca_dir}/trustedCAs.pem
         system -W ${destroot}${tls_ca_dir} \
             "${prefix}/share/apple-pki-bundle/bin/pems_not_in_pemfile.sh \
                     ${prefix}/share/curl/curl-ca-bundle.crt \
-                    trustedCAs.pem-temp \
-                        >> trustedCAs.pem"
-        delete ${destroot}${tls_ca_dir}/trustedCAs.pem-temp
+                    ${destroot}${tls_ca_dir}/trustedCAs.pem-temp \
+                        >> ${destroot}${tls_ca_dir}/trustedCAs.pem"
+        if { [variant_isset "user_pki_bundle"] \
+            && [file exists ${user_pki_bundle}]} {
+            delete  ${destroot}${tls_ca_dir}/trustedCAs.pem-temp
+            copy    ${destroot}${tls_ca_dir}/trustedCAs.pem \
+                    ${destroot}${tls_ca_dir}/trustedCAs.pem-temp
+            system -W ${destroot}${tls_ca_dir} \
+                "${prefix}/share/apple-pki-bundle/bin/pems_not_in_pemfile.sh \
+                    ${user_pki_bundle} \
+                    ${destroot}${tls_ca_dir}/trustedCAs.pem-temp \
+                        >> ${destroot}${tls_ca_dir}/trustedCAs.pem"
+        }
+        delete      ${destroot}${tls_ca_dir}/trustedCAs.pem-temp
+
+        xinstall -m 0664 \
+            ${destroot}${tls_ca_dir}/trustedCAs.pem \
```

Comment:
The copy-pasta relative to `${destroot}${tls_ca_dir}`, `${prefix}/share/apple-pki-bundle`, etc, makes my brain hurt!  :-D

Not necessary to fix ASAP - it's common for port authors to do this routinely - but a potential cleanup idea for the future.